packages/date-picker/src/vaadin-date-picker-mixin.js

```diff
@@ -264,6 +263,7 @@ export const DatePickerMixin = (subclass) =>
          */
         min: {
           type: String,
+          value: '',
```

Comment:
I guess we can now update types for `min` and `max` to use `string` instead of `string | undefined`?